demisto_sdk/commands/common/schemas/dashboard.yml

```diff
@@ -32,6 +32,11 @@ mapping:
     type: seq
     sequence:
       - include: layout_schema
+  marketplaces:
+    type: seq
+    sequence:
+      - type: str
+        enum: [ 'xsoar', 'marketplacev2', 'xpanse' ]
```

Comment:
```suggestion
```
Please open an issue to validate that all `marketplaces` field items are valid (according to the pythonic MarketplaceVersions enum).
1. We already define supported marketplaces in content items' parsers.
2. We want to have as least places to update as possible when we add support in a new marketplace - as an example you can see that although XPANSE supports incident fields (see incident field parser), it's missing in the schema.

Please remove the enum from other schemas as well, and add the marketplaces field where it's missing.